src/coefficients/stdfunction_coefficient.cpp

```diff
@@ -11,18 +11,33 @@
 
 namespace serac {
 
-StdFunctionCoefficient::StdFunctionCoefficient(std::function<double(mfem::Vector&)> func) : func_(func) {}
+StdFunctionCoefficient::StdFunctionCoefficient(std::function<double(mfem::Vector&)> func)
+    : func_([=](mfem::Vector& v, double /* t */) { return func(v); }), is_time_dependent_(false)
+{
+}
+StdFunctionCoefficient::StdFunctionCoefficient(std::function<double(mfem::Vector&, double)> func)
+    : func_(func), is_time_dependent_(true)
+{
+}
 
 double StdFunctionCoefficient::Eval(mfem::ElementTransformation& Tr, const mfem::IntegrationPoint& ip)
 {
   mfem::Vector transip(Tr.GetSpaceDim());
   Tr.Transform(ip, transip);
-  return func_(transip);
+  return func_(transip, GetTime());
 }
 
 StdFunctionVectorCoefficient::StdFunctionVectorCoefficient(int                                               dim,
                                                            std::function<void(mfem::Vector&, mfem::Vector&)> func)
-    : mfem::VectorCoefficient(dim), func_(func)
+    : mfem::VectorCoefficient(dim),
+      func_([=](mfem::Vector& v, mfem::Vector& w, double /* t */) { return func(v, w); }),
+      is_time_dependent_(false)
+{
+}
+
+StdFunctionVectorCoefficient::StdFunctionVectorCoefficient(
+    int dim, std::function<void(mfem::Vector&, mfem::Vector&, double)> func)
+    : mfem::VectorCoefficient(dim), func_(func), is_time_dependent_(true)
```

Comment:
This PR predated our transition to mfem 4.2, so I used the implementation we already had. However, it would make more sense to use the mfem native options now that they're available.

Reply:
Yep, it wasn't merged in until just yesterday.